### DISCUSSION TITLE
NativeScript 8 with Angular Cli

### DISCUSSION THREAD

**Discussion by tpersaud** [NativeScript/NativeScript#9901] [General]
Is there a way to use Angular Cli 13 to generate my components using NativeScript 8?

**Comment by NathanWalker** [comment]
@tpersaud We recommend using Angular component generation alongside NativeScript using [nx workspaces](https://nx.dev/) with [xplat](https://nstudio.io/xplat/getting-started/getting-started) which contains all the NativeScript Angular generators, for example:
```
nx g @nstudio/angular:component --name=account --platforms=nativescript
```
You can see this thread which discusses more of the generator options:
https://github.com/nstudio/xplat/issues/275#issuecomment-877835227
